frontend/src/lib/assets/GitHubImage.svelte

```diff
@@ -0,0 +1,21 @@
+<script>
+  export let className = '';
+  export let height = '96';
+  export let width = '98';
+  export let viewBox = '0 0 98 96';
```

Comment:
This is legacy mode for passing props

In runes mode this is preferred
```suggestion
  const { className, height = '96', width = '98', viewBox = '0 0 98 96' } = $props()
```

Or better

```suggestion
  const { className, height = '96', width = '98', viewBox = '0 0 98 96', ...rest }: SvelteHTMLElements['svg'] = $props()
```

and pass `{...rest}` to the SVG too